app/recurring_expenses/views.py

```diff
@@ -123,6 +123,52 @@ def create_recurring_expense(request):
     return render(request, "recurring_expenses/new.html", {"form": form})
 
 
+@login_required
+def export_recurring_expenses(request):
+    if "csv" in request.GET["format"]:
+        filename = "recurring_expenses.csv"
+        response = HttpResponse(
+            content_type="text/csv",
+            headers={"Content-Disposition": 'attachment; filename="' + filename + '"'},
+        )
+        writer = csv.writer(response)
+        writer.writerow(
+            [
+                "Active",
+                "Particulars",
+                "Amount",
+                "Currency",
+                "Amount NZD",
+                "Period (Months)",
+            ]
+        )
+        #
+        # Get expenses and amount in NZD
+        # Copied from class IndexView, maybe this should be a function?
+        # Exporting each expense to csv within this though so we don't
+        # have to run the for loop twice
+        #
+        fx_rates = get_fxRate_nzd()
+        expenses = RecurringExpense.objects.all()
+        for expense in expenses:
+            amount_nzd = (1 / fx_rates[expense.currency]) * float(expense.amount)
+            expense.amount_nzd = amount_nzd
```

Comment:
In the very near future we should extract this logic to a resuable function as it will end up in 3 places by the time we do JSON export, CSV export and the HTML view.

Reply:
Doesn't need to be done in this PR though. Just an observation

Reply:
Hence the comments about :D